docs/bn/week06/06-1.md

```diff
@@ -0,0 +1,186 @@
+
+
+<!--**Limitation**-->
+
+<!--Back in the 2000s, computation resources were restricted. The robot was able to process around 1 frame per second, which means it would not be able to detect a person that walks in its way for a whole second before being able to react. The solution for this limitation is a **Low-Cost Visual Odometry** model. It is not based on neural networks, has a vision of ~2.5m but reacts quickly.-->
+
+**সীমাবদ্ধতা**
+২০০০ এর দশকে, গণনার সংস্থানগুলি সীমাবদ্ধ ছিল। রোবট প্রতি সেকেন্ডে প্রায় 1 ফ্রেম প্রক্রিয়া করতে সক্ষম হয়েছিল, যার অর্থ এটি প্রতিক্রিয়া জানাতে সক্ষম হওয়ার আগে পুরো এক সেকেন্ডের জন্য কোনও ব্যক্তিকে সনাক্ত করতে সক্ষম হবে না যিনি পুরো দ্বিতীয় সেকেন্ডে  রোবটের পথে চলেছেন । এই সীমাবদ্ধতার সমাধান হল **কম দামের ভিজ্যুয়াল ওডোমেট্রি মডেল**। এটি নিউরাল নেটওয়ার্কগুলির উপর ভিত্তি করে নয়, ~ 2.5 m এর দর্শন রয়েছে তবে দ্রুত প্রতিক্রিয়া দেখায়।
+
+
+### Scene Parsing and Labelling
+
+<!--In this task, the model outputs an object category (buildings, cars, sky, etc.) for every pixel. The architecture is also multi-scale (Figure 6).-->
+
+##দৃশ্য বিশ্লেষণ  এবং লেবেলিং
+এই কার্যক্রমে, মডেলটি প্রতিটি পিক্সেলের জন্য একটি বস্তুর বিভাগ (বিল্ডিং, গাড়ি, আকাশ, ইত্যাদি) আউটপুট দেয়। মডেলের কাঠামোটি  বহু-স্কেল মাত্রায় গঠিত  (চিত্র 6)। 
+
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-1/VpVbkl5.jpg" style="zoom: 30%; background-color:#DCDCDC;"/><br>
+<b>Figure 6:</b> Multi-scale CNN for scene parsing
+</center>
+
+<!--Notice that if we back project one output of the CNN onto the input, it corresponds to an input window of size $46\times46$ on the original image at the bottom of the Laplacian Pyramid. It means we are **using the context of $46\times46$ pixels to decide the category of the central pixel**.-->
+লক্ষ্য করুন যে আমরা যদি সিএনএন-এর একটি আউটপুট ইনপুটটিতে প্রজেক্ট করি তবে এটি ল্যাপ্লাসিয়ান পিরামিডের নিচে মূল চিত্রের 46x46 আকারের একটি ইনপুট উইন্ডোটির সাথে মিল প্রদর্শন করে । এর অর্থ আমরা **কেন্দ্রীয় পিক্সেলের বিভাগ নির্ধারণ করতে 46 × 46 পিক্সেলের প্রসঙ্গটি ব্যবহার করছি**।
+
+<!--However, sometimes this context size is not enough to determine the category for larger objects.-->
+
+<!--**The multiscale approach enables a wider vision by providing extra rescaled images as  inputs.** The steps are as follows:-->
+**মাল্টিস্কেল পদ্ধতিটি অতিরিক্ত উদ্ধারকৃত চিত্রগুলি ইনপুট হিসাবে সরবরাহ করে একটি বিস্তৃত দৃষ্টি সক্ষম করে**। নিম্নরূপ পদক্ষেপ অনুসরণ করা হয় ঃ 
+
+1. Take the same image, reduce it by the factor of 2 and a factor of 4, separately.
+2. These two extra rescaled images are fed to **the same ConvNet** (same weights, same kernels) and we get another two sets of Level 2 Features.-->
+3. **Upsample** these features so that they have the same size as the Level 2 Features of the original image.
+4. **Stack** the three sets of (upsampled) features together and feed them to a classifier.
```

Comment:
These should be commented out?
```suggestion
<!--
1. Take the same image, reduce it by the factor of 2 and a factor of 4, separately.
2. These two extra rescaled images are fed to **the same ConvNet** (same weights, same kernels) and we get another two sets of Level 2 Features.
3. **Upsample** these features so that they have the same size as the Level 2 Features of the original image.
4. **Stack** the three sets of (upsampled) features together and feed them to a classifier.
-->
```

Reply:
I have commented out the portion.